tests/pytests/test_json.py

```diff
@@ -462,24 +462,6 @@ def testMultiValueTag_Recursive_Decent(env):
     env.expect('FT.SEARCH', 'idx', '@name:{foo}').equal(res)
     env.expect('FT.SEARCH', 'idx', '@name:{bar}').equal(res)
 
-@no_msan
-def testMultiValueVector(env):
```

Comment:
@GuyAv46 Suggest to keep some minimal test which fails on multi-value, which is not an plain array, so it is clear it is not supported and it is covered.
I also suggest to add a comment about this to the PR description.